### ISSUE TITLE
ToInt keeps empty string instead of changing to NaN

### ISSUE THREAD

**Issue by lazarljubenovic** [typestack/class-sanitizer#13]
The `ToInt` sanitizer doesn't take into account empty string. Looks like a `validator.js` upstream issue, but since it's currently locked on version 5, which is five major versions behind, it's worth testing if a simple bump will work first. Otherwise this issue should be taken there.

```ts
class Query {
  @ToInt()
  readonly page: number
}

const q = new Query() as any
q.page = ''
sanitize(q)

console.log(q) // { page: '' }
```

Reproduction: https://stackblitz.com/edit/class-sanitizer-to-int-issue?file=index.ts

This happens with query parameters sent as `&page=`.

**Comment by YouriT** [comment]
addressed in https://github.com/swanest/class-sanitizer/commit/a31bdb903f6e27b66cd3b58f60d25bc8ce1ae125

**Comment by NoNameProvided** [comment]
Fixed in v1.0.1
